waku/v2/protocol/rln/waku_rln_relay.go

```diff
@@ -131,30 +132,30 @@ func (rlnRelay *WakuRLNRelay) ValidateMessage(msg *pb.WakuMessage, optionalTime
 	if int64(math.Abs(float64(gap))) > maxEpochGap {
 		// message's epoch is too old or too ahead
 		// accept messages whose epoch is within +-MAX_EPOCH_GAP from the current epoch
-		rlnRelay.log.Debug("invalid message: epoch gap exceeds a threshold", zap.Int64("gap", gap))
+		rlnRelay.log.Info("invalid message: epoch gap exceeds a threshold", zap.Int64("gap", gap))
```

Comment:
Should probably be Debug. (This could be a frequent scenario, so no need for it to be an Error)